### DISCUSSION TITLE
Zirnox getTemp(), getPressure() show random numbers

### DISCUSSION THREAD

**Discussion by AceOfZpades** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1482] [Q&A] [answered]
![2024-05-09_17 04 50](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/168376068/7acbff10-f01b-4c27-b600-7824d45c27c9)
Above Screenshot shows the code
![2024-05-09_17 07 08](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/168376068/ea6db143-ff10-433c-a00c-19874210010b)
Above screenshot shows the temperature/pressure the code prints, when in actuality the temperature is 20C and 10 Bars

**Comment by gmhs022** [comment, marked as answer]
The values are not measured in C or Bar, they're measured in abstract units found in the Zirnox tile entity code.
I've made some simple functions to convert them here:

```lua
local temp_c = zirnox.getTemp() * 1e-5 * 780 + 20 -- abstract temperature unit into *C
local pres_b = zirnox.getPressure() * 1e-5 * 30.0 -- abstract pressure unit into Bar
```

**Reply by AceOfZpades** [reply]
Thanks for the help, this worked.
